src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -98,15 +101,7 @@ public sealed class UIActionEngine
                         new GatherCache(),
                         sourceCacheContext);
 
-                    return GetActionsAsync(
-                        uiService,
-                        projects,
-                        userAction,
-                        uiService.RemoveDependencies,
-                        uiService.ForceRemove,
-                        resolutionContext,
-                        projectContext: uiService.ProjectContext,
-                        token: token);
+                    throw new NotImplementedException();
```

Comment:
assume this gets fixed with install/uninstall work.

Reply:
Yep